.tekton/manifests/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - namespace.yaml
+  - repository.yaml
+  - https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.9/git-clone.yaml
+  - https://raw.githubusercontent.com/tektoncd/catalog/main/task/github-open-pr/0.2/github-open-pr.yaml
```

Comment:
The both tasks `git-clone.yaml` and `github-open-pr` are build-in ClusterTask in Openshift-pipeline. Do we have to add them?

Reply:
It wouldn't work before I added them. Based on the conversations we had, I was expecting the ClusterTask would be retrieved automatically from the tekton catalog, but that is not the behavior I experienced.

Reply:
I just realized that Pipelines as Code support fetching remote tasks or pipeline from a remote location with annotations on PipelineRun. For example
```
pipelinesascode.tekton.dev/task: "[git-clone, github-open-pr]"
```

Reply:
@xinredhat Fixed